lib/nylas/model.rb

```diff
@@ -64,7 +64,6 @@ def update(**data)
         path: resource_path,
         query: query_params
       )
-      true
```

Comment:
Not sure if it's a good idea to change response here as existing apps might be check if `update` returns `true` or not

Reply:
Yeah I know that this may be an issue for pre-existing apps, but without making this change, then we won't be able to know the version number in the `draft` class:

```
response = super
return unless response[:version]

self.version = response[:version]
```

`response` would just be `true` and not a JSON. The alternative is that we check if it's `true` and increment the version number ourselves. But I'm not a fan of making the SDK 'predict' an attribute locally in case the Nylas API changes in the future. Let me know what you think.

Reply:
Agreed, I like your approach much better.